src/libraries/analytics/firebaseAnalytics.js

```diff
@@ -1,11 +1,15 @@
 import analytics from '@react-native-firebase/analytics';
 
+import getFeatureLoggerInstance, {EFeatureLogFlag} from '../../utils/log/FeatureLog';
+
+const {featLog} = getFeatureLoggerInstance(EFeatureLogFlag.firebaseUtils);
+
 const FirebaseAnalytics = () => ({
   logEvent(eventName, param) {
     analytics().logEvent(eventName, param);
     if (__DEV__) {
-      console.log('Log event: ', eventName);
-      console.log('log Event params: ', param);
+      featLog('Log event: ', eventName);
+      featLog('log Event params: ', param);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of `console.log` with `featLog` is a good practice for production code. It provides a more structured way of logging and can be easily turned off in production if needed. However, make sure that the `featLog` function has been thoroughly tested and handles all possible edge cases, such as null or undefined parameters.
```diff
     analytics().logEvent(eventName, param);
     if (__DEV__) {
      featLog('Log event: ', eventName);
      featLog('log Event params: ', param);
+        // Ensure featLog can handle null or undefined parameters
+        featLog('Test log with undefined: ', undefined);
+        featLog('Test log with null: ', null);
     }
```
This will help you catch any potential issues early on during development.


<!-- This is an auto-generated comment by OSS CodeRabbit -->